ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/maintenance.j2

```diff
@@ -0,0 +1,52 @@
+{% if maintenance is arista.avd.defined %}
+
+## Maintenance
+
+### Maintenance defaults
+
+Default maintenance bgp profile: **{{ maintenance.default_bgp_profile | arista.avd.default('Default') }}**
+
+Default maintenance interface profile: **{{ maintenance.default_interface_profile | arista.avd.default('Default') }}**
+
+Default maintenance unit profile: **{{ maintenance.default_unit_profile | arista.avd.default('Default') }}**
+
+### Maintenance profiles
+
+| BGP profile | Initiator route-map
+| -------- | ----- |
+{%     for bgp_profile in maintenance.bgp_profiles | arista.avd.natural_sort %}
+| {{ bgp_profile }} | {{ maintenance.bgp_profiles[bgp_profile].initiator.route_map_inout }} |
+{%     endfor %}
+
+| Interface profile | Rate monitoring load interval | Rate monitoring threshold in/out |
+|-------------------|-------------------------------|------------------|
+{%     for interface_profile in maintenance.interface_profiles | arista.avd.natural_sort %}
+| {{ interface_profile -}}
```

Comment:
to make this more readable, please set some temporary variables like `set row_load_interval = maintenance.interface_profiles[interface_profile].rate_monitoring.load_interval | arista.avd.default(60)` and use those shorter variables in the table fields, which can then stay on a single line. Same for "Maintenance Units" table.
Also please add a unit to the heading. Like `(s)` for interval and `(kbps)` for threshold to help the reader.